### ISSUE TITLE
Change `complete` to `output`

### ISSUE THREAD

**Issue by Pwntus** [Zebreus/replicate-api#4]
Hi,

I think `complete` is supposed to be `output` here, as per Replicate documentation:
https://github.com/zebreus/replicate-api/blob/master/src/predict.ts#L33

```
start: immediately on prediction start
output: each time a prediction generates an output (note that predictions can generate multiple outputs)
logs: each time log output is generated by a prediction
completed: when the prediction reaches a terminal state (succeeded/canceled/failed)
```

**Comment by zebreus** [comment]
:eagle: Thanks, I will fix it right now.

**Comment by zebreus** [comment]
Fixed in v0.4.1

**Comment by Pwntus** [comment]
Thank you!
